### PR TITLE
Add wochenende_plot.sh update.Minor changes, fixed wochenende_preprocess bug

### PR DESCRIPTION
**Pull request by colindaven**

### CODE REVIEW

File: wochenende_postprocess.sh [hunk 1]
```diff
@@ -58,18 +61,32 @@ echo "INFO: Starting Wochenende_postprocess"
 echo "INFO: Current directory" $bamDir
 sleep 3
```
Comment (B1T0) [inline wochenende_postprocess.sh:62]: Just a general comment: Some of the sleep timers seem to be not fully necessary like this (at least from an outside perspective) and might cause an unpleasant experience due to somewhat useless waiting times. However, if they are definitely needed, of course, we should keep them

File: plots/wochenende_plot.py [hunk 2]
```diff
@@ -2,526 +2,823 @@
 # coding: utf-8
```
Comment (B1T0) [inline plots/wochenende_plot.py:2]: Are these the same changes as in #107?

**Review by B1T0 — APPROVED** [review]
Looks good to me 👌 I just had two general comments. (I don't know where they show up. If you can't see them just mention me in another comment so I will get a notification)

**Review by poer-sophia — APPROVED** [review]

File: plots/wochenende_plot.py (hunk 2 above)
Comment (colindaven) [reply plots/wochenende_plot.py:2]: Yes, Sophia forked and pulled to dev, these are those PR and commits (with further modifications by Colin in dev) now being pulled to master. 
@poer-sophia is this correct ? Please OK this point, I think it's fine.

File: wochenende_postprocess.sh (hunk 1 above)
Comment (colindaven) [reply wochenende_postprocess.sh:62]: Yes, there were problems with NFS write errors when the cluster was under load, so I needed to put in the timers. Things are better tested now, but the timers don't really hurt. What hurts is having to repeatedly start the postprocess script when something goes wrong. We can remove them later or include these as an option.

File: plots/wochenende_plot.py (hunk 2 above)
Comment (B1T0) [reply plots/wochenende_plot.py:2]: I just checked, you're right. I did not see that it was a PR to dev. Sorry for the confusion

**Pull request merged**